scripts/test/test-backend.sh

```
@@ -108,16 +136,21 @@ test_remote_backend() {
-    if ping -c 2 <NODE_B_IP> >/dev/null 2>&1; then
-        echo -e "${GREEN}✓${NC} ICMP ping to Node B (<NODE_B_IP>)"
+    if ping -c 2 "$NODE_B_HOST" >/dev/null 2>&1; then
+        echo -e "${GREEN}✓${NC} ICMP ping to Node B (${NODE_B_HOST})"
     else
         echo -e "${YELLOW}!${NC} ICMP ping blocked (may be normal)"
     fi
     
     # Test HTTP connectivity via SSH tunnel
     echo -e "${YELLOW}Creating SSH tunnel for HTTP test...${NC}"
-    ssh -o StrictHostKeyChecking=no -o ConnectTimeout=5 -f -N -L 8082:localhost:8080 root@<NODE_B_IP> 2>/dev/null
+    ssh $SSH_OPTS -f -N -L 8082:localhost:8080 "${NODE_B_USER}@${NODE_B_HOST}" 2>/dev/null
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)